### DISCUSSION TITLE
[gatsby-plugin-google-gtag] - Add 'includeInDevelopment' flag in options.

### DISCUSSION THREAD

**Discussion by abohannon** [gatsbyjs/gatsby#29871] [Ideas / Feature Requests]
## Summary

Allows gtag to be loaded in development. Currently, it is only loaded in production.

### Basic example

Add an `includeInDevelopment` flag similar to `gatsby-plugin-google-tagmanager`.
```
 {
      resolve: `gatsby-plugin-google-gtag`,
      options: {
        includeInDevelopment: true,
        trackingIds: ['GA_ID'],
      },
    }
```

### Motivation

We're currently revamping our entire analytics architecture at work which is fairly complex and need to test events locally before deploying to production. Allowing gtag to load in development will help tremendously and save us from needing to build every time we want to test.

We're currently using a modified fork of this plugin with the `includeInDevelopment` flag added. I'm happy to submit a PR if the community thinks this is a valuable contribution.

**Comment by gtsop** [comment]
@abohannon I need this as well. Can you please share your solution?
